community.md

```diff
@@ -37,6 +53,24 @@ We’re happy to help! Come engage with us on any channel that works for you:
    </div>
 </div>
 
+## Report issues
+
+Hit a bump on the road? Report an issue in the relevant repo out of the GitHub repos listed below. We want to fix all bugs!
+
+   - compiler, runtime, standard library, or tooling: <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo.
+   - language specification: <a href="https://github.com/ballerina-platform/ballerina-spec/issues">ballerina-spec</a> repo.
+   - website: <a href="https://github.com/ballerina-platform/ballerina-dev-website/issues">ballerina-dev-website</a> repo. 
+   - security flaw: send an email to security@ballerina.io. For details, see the <a href="https://ballerina.io/security/">security policy</a>.

```

Comment:
Shouldn't we start with an upper case letter in each bullet point?

Reply:
Done.